File: Temporada02/Episodio01/C/ejemplo.c
/******************************************************************************

                            Online C Compiler.
                Code, Compile, Run and Debug C program online.
Write your code in this editor and press "Run" button to compile and execute it.

*******************************************************************************/

#include <stdio.h>
char* antes = "Antes";
char* palabras[3] = {"Hola", " ", "mundo"};
char* despues = "Despues";

int main()
{
    for(int i=0;i<3;i++)
        printf("%s", palabras[i]);

    return 0;
}

File: Temporada02/Episodio01/Pascal/Ejemplo.pas
{

Online Pascal Compiler.Code, Compile,
    Run and Debug Pascal program online.Write your code in this editor and
    press "Run" button to execute it. $RANGECHECKS ON
}

program Hello;

var numeros: array[1..3] of integer;
var i: integer;

begin
    numeros[1]:= 100;
    numeros[2]:= 200;
    numeros[3]:= 300;

    for i:= 1 to 3 do
        writeln ('numeros[', i, '] = ', numeros[i]);
end.
